src/node/rpc/memberfrontend.h

```diff
@@ -681,10 +681,16 @@ namespace ccf
             HTTP_STATUS_BAD_REQUEST, "Submitted state digest is not valid");
         }
 
-        ma_view->put(
-          args.caller_id,
-          MemberAck(sig_view->get(0)->root, signed_request.value()));
-
+        const auto s = sig_view->get(0);
+        if (!s)
```

Comment:
Just to confirm that my understanding is right: this would happen if a member acks before the first signature/pre-prepare has been emitted?

Reply:
Yes